doc/api/models/schemas/CreateNodeCollateral.yml

```diff
@@ -0,0 +1,8 @@
+type: object
+properties:
+  stake:
+    $ref: 'NodeCollateral.yml'
+  tokenLockRef:
+    $ref: 'Hash.yml'
+  collateralRef:
```

Comment:
This reference actually needs an ordinal -- the tokenLockRef doesn't, but there needs to be a unique type with a hash/ordinal for this one unfortunately